src/component/modal/bekreft-send-vedtak-modal.tsx

```diff
@@ -20,6 +25,18 @@ export default function BekreftSendVedtakModal(props: VedtakSendtModalProps) {
 				<Heading level="1" size="medium">
 					Send vedtaksbrev
 				</Heading>
+				<div className="bekreft-send-vedtak-modal-personinfo">
+					<BodyShort
+						weight={'semibold'}
+					>{`${formateStringInUpperAndLowerCase(navn.fornavn)} ${formateStringInUpperAndLowerCase(navn.etternavn)}`}</BodyShort>
```

Comment:
Skal vi vise mellomnamn også, dersom personen har det?

Reply:
Det er en liten varselmodal, tenker det holder med for- og etternavn pluss fnr.